spec/factories/payment_requests.rb

```diff
@@ -11,5 +11,15 @@
     payment_status { "pending" }
     ready_for_payment_processing { true }
     payment_attempts { 0 }
+
+    transient do
+      invoices { [] }
+    end
+
+    after(:create) do |payment_request, evaluator|
+      evaluator.invoices.each do |invoice|
+        PaymentRequest::AppliedInvoice.create!(payment_request:, invoice:)
+      end
+    end
```

Comment:
doesn't rails automatically create the applied invoice join records when assigning invoices to the payment request record?